tests/fast-integration/README.md

```diff
@@ -46,6 +46,25 @@ npm install
 npm test
 ```
 
+
+#### To run eventing specific tests locally, follow these steps:
```

Comment:
```suggestion
To run Eventing-specific tests locally, follow these steps:
```

Making it consistent since the previous [section](https://github.com/kyma-project/kyma/blob/69832fb62d6ac149d46b0f3336a4cf7d857afaf7/tests/fast-integration/README.md#usage) `To run tests locally...` isn't bolded.